### PR TITLE
Implement the `customRegexFilter` function 

### LINKED ISSUE #5: Implement the `customRegexFilter` function
Below is the code located at `src/moderation/customFilter.ts`

```ts
import { Message, MessageType } from "discord.js";
import { Guild } from "../configs/guild";

function customWordFilter(message: Message, guildConfig: Guild) {
    for (const words of message.content.split(" ")) {
        if (guildConfig.moderation.customWordFilter.includes(words)) {
            try { message.delete() }
            // Incase bot does not have permission to delete
            catch { return }

            try { message.channel.send(`<@${message.author.id}>, the message you sent contained content not allowed on this server!`) }
            catch { return }
        }
    }
}

function customRegexFilter(message: Message, guildConfig: Guild) {

}

export { customWordFilter, customRegexFilter }
```

As you can see, the customRegexFilter function is empty

I will need you to use the `guildConfig.moderation.customRegexFilter` which will return a String[] object containing a list of regular expressions in string format (eg "/https:\/\/discord\.gg\/([a-zA-Z0-9-]{5,})/;")

You will need to use something like new RegExp(string) or whatever you come up with and then you will have to check it against the messages, similar to what I did in the first function. You can then add it to the `src/moderation/index.ts` and test it on the test server

### PR DESCRIPTION
**Pull request by hackermondev**
#5

### CODE REVIEW

File: src/moderation/customFilter.ts [hunk 1]
```diff
@@ -15,7 +15,19 @@ function customWordFilter(message: Message, guildConfig: Guild) {
 }
 
 function customRegexFilter(message: Message, guildConfig: Guild) {
-
+    for (const words of message.content.split(" ")) {
```
Comment (Nissoco23) [inline src/moderation/customFilter.ts]: I do not think you need to split the message when testing a regex against a sentence, you should be able to remove it for higher performance and code efficiency

**Review by Nissoco23 — CHANGES_REQUESTED** [review]
I do

File: src/moderation/customFilter.ts (hunk 1 above)
Comment (hackermondev) [reply src/moderation/customFilter.ts]: oh right

**Review by Nissoco23 — APPROVED** [review]
Everything looks good. Nice job

**Pull request merged**